phorge/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+services:
+  phorge:
+    build:
+      context: phorge-container
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    ports:
+      - "8030:80"
+    depends_on:
+      - phorge-mysql
+    links:
+      - phorge-mysql
+    environment: 
+      - MYSQL_HOST=phorge-mysql
+      - MYSQL_PORT=3306
+      - MYSQL_USER=root
+      - MYSQL_PASSWORD=${MYSQL_ROOT_PASSWORD}
+      - BASE_URI=${BASE_URI}
+      - SMTP_SERVER=${SMTP_SERVER}
+      - SMTP_PORT=${SMTP_PORT}
+      - SMTP_USER=${SMTP_USER}
+      - SMTP_PASSWORD=${SMTP_PASSWORD}
+      - SMTP_PROTOCOL=${SMTP_PROTOCOL}
+    volumes: 
+      - ./ph-repo:/var/repo
+    healthcheck:
+      test: ["CMD", "curl", "-H", "Host: ${BASE_URI}", "http://localhost:80/status"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  phorge-mysql:
+    image: mariadb:10.6.14
+    restart: unless-stopped
+    volumes:
+      - ./database-volume:/var/lib/mysql
```

Comment:
Please just use named volumes for data - they will be automatically backed up then